com.w1hkj.fldigi.yml

```diff
@@ -0,0 +1,53 @@
+app-id: com.w1hkj.fldigi
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
```

Comment:
No settings seem to be saved. You should add
```
   - --persist=.fldigi
   - --persist=.fltk
   - --persist=.nbems
```
to the finish-args to make these directories persistent.

Reply:
ok

Reply:
ahhh I just realized when I provided it on my machine .. thanks .. correcting

Reply:
the same for flcluster .. because it doesn't save